kotlinx-coroutines-debug/build.gradle

```diff
@@ -35,5 +39,7 @@ shadowJar {
     classifier null
     // Shadow only byte buddy, do not package kotlin stdlib
     configurations = [project.configurations.shadowDeps]
-    relocate 'net.bytebuddy', 'kotlinx.coroutines.repackaged.net.bytebuddy'
+    relocate('net.bytebuddy', 'kotlinx.coroutines.repackaged.net.bytebuddy') {
+        exclude 'net.bytebuddy.agent'
```

Comment:
please leave a comment why we exclude `agent` here

Reply:
Good catch: it seems to work fine as is, even on JDK 8. I no longer exclude `agent`.